From: Ichthyostega <prg@ichthyostega.de>
Date: Fri, 28 Nov 2025 20:04:11 +0100
Subject: Disable known fragile tests for DEB package

Some of the scheduler tests are notoriously fragile,
since they verify observable timings.

I have not yet found a better way to verify their objectives
and I could not determine timings that work reliably even directly
after a full-load package build.

Maybe we need to relegate those tests into a special testsuite anyway?
---
 tests/15library.tests   | 2 +-
 tests/32scheduler.tests | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/15library.tests b/tests/15library.tests
index 262d6d2..0f56260 100644
--- a/tests/15library.tests
+++ b/tests/15library.tests
@@ -449,7 +449,7 @@ return: 0
 END
 
 
-TEST "Instrumentation: concurrent calls" IncidenceCount_test <<END
+PLANNED "Instrumentation: concurrent calls" IncidenceCount_test <<END
 return: 0
 END
 
diff --git a/tests/32scheduler.tests b/tests/32scheduler.tests
index 7687236..b77c4a8 100644
--- a/tests/32scheduler.tests
+++ b/tests/32scheduler.tests
@@ -56,7 +56,7 @@ END
 
 
 
-TEST "Scheduler Performance" SchedulerStress_test quick <<END
+PLANNED "Scheduler Performance" SchedulerStress_test quick <<END
 return: 0
 END
 
@@ -68,6 +68,6 @@ END
 
 
 
-TEST "Worker Thread Service" WorkForce_test <<END
+PLANNED "Worker Thread Service" WorkForce_test <<END
 return: 0
 END
